### PR TITLE
ATLEDGE-352 fix dfu-util -d flag in upload script

### PR DESCRIPTION
**Pull request by bbaltz505**
Signed-off-by: Brian Baltz brian.a.baltz@intel.com

### CODE REVIEW

**Comment by bbaltz505** [comment]
Moved to public repo. @kmsywula please review.

File: clupload/cluploadArduino101_linux.sh [hunk 1]
```diff
@@ -24,17 +24,17 @@ echo "BIN FILE" $bin_file_name
 DFU=$fixed_path/dfu-util
 echo "wating for Arduino 101 device... "
 COUNTER=0
-f=`$DFU -l -d 8087:0ABA | grep sensor_core | cut -f 1 -d ' '`
+f=`$DFU -l -d ,8087:0ABA | grep sensor_core | cut -f 1 -d ' '`
```
Comment (calvinatintel) [inline clupload/cluploadArduino101_linux.sh:27]: Is it normal to have a space between -d and 8087?

Comment (bbaltz505) [inline clupload/cluploadArduino101_linux.sh:27]: I don't think it matters. I've seen it implemented both ways. The comma separates the PID/VID and the DFU PID/VID:
 -d --device <vendor>:<product>[,<vendor_dfu>:<product_dfu>]

Comment (calvinatintel) [inline clupload/cluploadArduino101_linux.sh:27]: Ok, as long as it works. +1 from me.

Comment (SidLeung) [inline clupload/cluploadArduino101_linux.sh:27]: It works either way, -d8087 or -d 8087, on Windows.

**Pull request merged**